databricks/koalas/indexes.py

```diff
@@ -2498,6 +2518,78 @@ def __getattr__(self, item: str) -> Any:
                 return partial(property_or_func, self)
         raise AttributeError("'MultiIndex' object has no attribute '{}'".format(item))
 
+    def _get_level_number(self, level):
+        """
+        Return the level number if a valid level is given.
+        **this is an internal method**
+        """
+        count = self.names.count(level)
+        if (count > 1) and not isinstance(level, int):
+            raise ValueError("The name %s occurs multiple times, use a level number" % level)
+        try:
+            level = self.names.index(level)
```

Comment:
I think you can use if-else instead of relying on try-catch.

Reply:
Thanks @HyukjinKwon , Fixed!